### PR TITLE
Updating documentation on 0.27-maintenance to respect ctest

### PR DESCRIPTION
**Pull request by clanmills**
Initial changes to documentation to support CTest.  I intend to integrate changes from main/README.md in this PR.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ $ cd ~/gnu/github/exiv2  # location of the project code
 $ mkdir build && cd build
 $ cmake .. -DCMAKE_BUILD_TYPE=Release
 $ cmake --build .
-$ make tests
+$ make test
```
Comment (kmilos) [inline README.md:74]: I suggest we use the same `ctest` command used in CI - there should be no explicit dependence on autotools.

Comment (clanmills) [reply README.md:74]: I agree.  Let's avoid talking about make and autotools magic.

**Comment by clanmills** [comment]
@kmilos.  I'm getting 100% test failure with the Visual Studio build.  I have to provide `-C Release` to get ctest to run at all.  I'll update README.md about that.

You mentioned something about 'git config'.  Is that something I need to deal with?  Any ideas what might be the matter?

I'm sure it's related to line-endings.

**Comment by kmilos** [comment]
No, that was very Cygwin specific.

I can take a look, but might be a while, the dad hat rarely comes off during the weekend...

**Comment by kmilos** [comment]
@clanmills The Windows CI seems to be ok? I see only codecov failing (no idea why)....

**Comment by clanmills** [comment]
Codecov is a "bloody old woman!".  It's complaining because code coverage has gone down _(by almost nothing)_.   We'll ignore that.

I'm sure I'll add more polish/fixes to this PR over the weekend.

**Review by kmilos — DISMISSED** [review]

**Review by kmilos — APPROVED** [review]

**Comment by clanmills** [comment]
This PR has been sitting in MSYS2 doing "setup" for 5 hours, so I cancelled it.  Everything else passed and this PR cannot be involved in setting up the build machine.

I looked at the documentation concerning codecov.  I think we can set threshold in codecov.yml to ignore negative code coverage.  The codecov documentation needs more careful investigation to decide what should be done.

I'm going to merge.

**Comment by kmilos** [comment]
Yeah, I've seen GHA jobs get stuck before on occasion...

**Pull request merged**